assets/src/edit-story/app/media/local/useContextValueProvider.js

```diff
@@ -156,17 +144,14 @@ export default function useContextValueProvider(reducerState, reducerActions) {
 
   const processor = useCallback(
     ({ mimeType, posterId, id, src, local }) => {
-      const process = async () => {
-        if (
-          allowedVideoMimeTypes.includes(mimeType) &&
-          !local &&
-          !posterId &&
-          id
-        ) {
-          await uploadVideoPoster(id, src);
-        }
-      };
-      process();
+      if (
```

Comment:
Why does change this from async to sync help the issue?

Reply:
This part does not fix the issue itself, but fixes a code quality issue outlined in #5541